### DISCUSSION TITLE
Manga Is All You Need

### DISCUSSION THREAD

**Discussion by tsubakittr** [NAIST-SE/AttractingContributors#56] [General]
# Any Manga is OK!
What is your favorite Manga?
Please feel free to make a pull request!!

#### https://github.com/tsubakittr/AttractingContributors-Manga/blob/main/README.md

**Comment by eedy8888** [comment]
added please help me
https://github.com/eedy8888/AttractingContributors

**Reply by tsubakittr** [reply]
I added your list!
thx :D

**Comment by vittunyuta-aom** [comment]
added please help me https://github.com/aommoaGitHub/SuperFavoriteSongs

**Reply by tsubakittr** [reply]
yeah, I added your list !
thx for your help :D

**Comment by manzoku23** [comment]
Added my favorite one.
Please help me 
https://github.com/Exe-dev/AttractingContributors-POKEMON

**Reply by tsubakittr** [reply]
thx!
Sure, I added Starmie to your list! 🥇

**Reply by manzoku23** [reply]
thx :D
